raster/src/main/scala/geotrellis/raster/ConstantTile.scala

```diff
@@ -243,6 +243,45 @@ object ConstantTile {
       case ct: FloatCells => FloatConstantTile.fromBytes(bytes, cols, rows, ct)
       case ct: DoubleCells => DoubleConstantTile.fromBytes(bytes, cols, rows, ct)
     }
+
+  /**
+   * Create a [[ConstantTile]] that is set to the nodata value of the given celltype or to the default nodata value if
+   * the celltype does not have a nodata. BitCells are set to 'false'.
+   *
+   * @param   t      The [[CellType]] of the new [[ConstantTile]].
+   * @param   cols   The number of columns that the new [[ConstantTile]] should have
+   * @param   rows   The number of rows that the new [[ConstantTile]] should have
+   * @return         The new [[ConstantTile]]
+   */
+  def empty(t: CellType, cols: Int, rows: Int): Tile = {
+    t match {
+      case _: BitCells => BitConstantTile(false, cols, rows)
+      case ct: ByteUserDefinedNoDataCellType => ByteConstantTile(ct.noDataValue ,cols, rows, ct)
```

Comment:
nice, super minor: it looks formatting is a bit off in the entire function; that space before the comma :D 

`(ct.noDataValue ,cols, rows, ct)` => `(ct.noDataValue, cols, rows, ct)`